src/module.ts

```diff
@@ -167,7 +167,7 @@ export default defineNuxtModule<ModuleOptions>({
             icon: 'logos-tailwindcss-icon',
             view: {
               type: 'iframe',
-              src: viewerConfig.endpoint
+              src: `/${viewerConfig.endpoint}/`.replace(/\/+/g, '/')
```

Comment:
By default `viewerConfig.endpoint` contains `_tailwind`, where we actually need a absolute path here as `/_tailwind/`